test-forms/src/main/resources/forms/pull_data.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<h:html xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml"
+    xmlns:jr="http://openrosa.org/javarosa" xmlns:odk="http://www.opendatakit.org/xforms"
+    xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+    xmlns="http://www.w3.org/2002/xforms">
+    <h:head>
+        <h:title>pull_data</h:title>
```

Comment:
It looks like we already have one form that uses pull data it's called `formWithExternalFiles` but it seems like it's not used so you can get rid of it.